public/docs/_examples/intro/dart/lib/hero_detail_component.html

```diff
@@ -0,0 +1,8 @@
+<!-- #docregion -->
+<hr>
+<h4>{{hero.name}} Detail</h4>
+<div>Id: {{hero.id}}</div>
+<div>Name:
```

Comment:
I did this before I got word back from you on the named `docregion`s. I'll go through all the current PR's I have out and update them.